[#2dmz4a6] Button Add To Cart

vstacked opened the pull request:
<!--
  Thanks for contributing!

  Provide a description of your changes below and a general summary in the title

  Please look at the following checklist to ensure that your PR can be accepted quickly:
-->

## Description
Adding button Add to Cart widget.
- add ui for portrait & landscape (base on scratch image on clickUp)

<!--- Describe your changes in detail -->

## Type of Change

<!--- Put an `x` in all the boxes that apply: -->

- [x] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [ ] 🗑️ Chore

## Evidence
<img src="https://user-images.githubusercontent.com/54944887/157878626-6c0d464a-9f9e-4a98-92ed-a209f1893565.png" height="300"> <img src="https://user-images.githubusercontent.com/54944887/157878647-e09756e4-0633-4496-8706-516615a77931.png" height="300">

vstacked committed:
feat(widget): :sparkles: add button add to cart

vstacked committed:
feat: :sparkles: add width parameter

nfach98 commented on lib/page/detail/widget/button_add_to_cart.dart:
```diff
@@ -0,0 +1,41 @@
+  Widget build(BuildContext context) {
+    final _radius = BorderRadius.circular(16);
+    return Container(
+      height: 70,
```
Height bisa dibuat mengikuti isi content, sehingga bisa fleksibel ketika font dikali textScaleFactor

vstacked committed:
refactor(button): :recycle: refactor button add to cart, add default param height & width, fix formatting

- change ElevatedButton to MaterialButton
- add default value param for width & height
- add condition height button for textScaleFactor
